### DISCUSSION TITLE
Navigate lint errors

### DISCUSSION THREAD

**Discussion by buhtz** [joaotavora/eglot#968] [Q&A]
I am not sure but assume that flake8 is used as linter by eglot and/or the language server behind.

In the buffer I see underlines and in the minbuffer the description for existing lint errors. I also see the ! in the modeline.

How can I navigate to the next lint error or display a list of lint errors in Emacs? I couldn't find a eglot command for this in M-x list.

**Comment by joaotavora** [comment]
Check out the Flymake manual: https://www.gnu.org/software/emacs/manual/html_mono/flymake.html#Finding-diagnostics

**Comment by hblwslyan-commits** [comment]
#wywytyt85@gmail.com
